src/views/Settings.vue

```diff
@@ -17,6 +17,14 @@
          </ion-select>
        </ion-item>
 
+       <ion-item>
+         <ion-icon :icon="locationOutline" slot="start" />
+         <ion-label>{{$t("Location")}}</ion-label>
+         <ion-select interface="popover" :placeholder="$t('facility location')" :value="currentFacilityLocation.locationSeqId" @ionChange="setFacilityLocation($event)">
+           <ion-select-option v-for="facilityLocation in (userProfile && userProfile.facilityLocations ? userProfile.facilityLocations : [])" :key="facilityLocation.locationSeqId" :value="facilityLocation.locationSeqId" >{{ facilityLocation.locationPath }}</ion-select-option>
```

Comment:
I don't think `facilityLocation` has property `locationPath` with it, let me know if I missed something.

Reply:
Yes Sir, we are not getting this field with API instead we are creating this field after getting response back from API.
https://github.com/hotwax/receiving/pull/89/files#:~:text=locationSeqId%3A%20location.locationSeqId,levelId%20%2B%20location.positionId